docs/use-cases.md

```diff
@@ -0,0 +1,59 @@
+
+This may sound ambitious and slightly confusing considering how many other projects exist in the VM scheduling space. By comparing KubeVirt to these other projects, it should be clear what KubeVirt’s scope is and where it fits in the ecosystem.
+
+# KubeVirt vs. Other Projects
+
+## Kubernetes
+
+Kubernetes is an open source project built to automate deployment and lifecycle management of containerized applications.
+
+KubeVirt is a drop in addon that can be applied to any to Kubernetes cluster that gives Kubernetes the ability to manage VMs.
+
+## OpenStack
+
+OpenStack is an open source IaaS platform built of various components to provide a wide range of features such as compute (VM scheduling), networking, block storage, object storage, ect.
+
+KubeVirt is a single component focused entirely on VM scheduling.  Network and block storage are something that KubeVirt consumes from other systems and not something KubeVirt attempts to define. Similar to the way Libvirt has become a standard for managing VMs on a local node across multiple projects, we hope to see KubeVirt become the widely adopted standard for scheduling VMs.  It is theoretically possible OpenStack could adopt KubeVirt for VM scheduling.
+
+## Nova
+
+Nova is the open source component of OpenStack responsible for VM scheduling. Nova provides an abstraction layer that allows multiple virtualization technologies to be utilized. This means Nova is maintaining drivers for technologies such as Xen, VMware, KVM, Hyper-V, and LXC.
+
+KubeVirt has made the choice to focus on KVM managed by Libvirt. Initially this may sound like it is limiting KubeVirt’s functionality, but in practice we see KVM and Libvirt emerging as the leading standard in open source virtualization. By focusing on this technology stack we aren’t limited by creating a one size fits all virtualization technology abstraction.  We can fully support a feature set that leverages all that Libvirt and KVM have to offer.
+
+## oVirt and RHEV
+
+oVirt is an open source virtualization management platform that provides VM scheduling, network and storage. One of the fundamental differences between oVirt and other similar projects like OpenStack is the requirement for infrastructure High Availability.
+
+Cloud infrastructure provided by OpenStack or public clouds like EC2 and GCE make the assumption that applications running on their platforms have resiliency built in.  oVirt is for the use cases where applications require strict HA guarantees at the infrastructure level. With oVirt, data integrity is enforced by node level fencing.
+
+This comes down to the “pets vs. cattle” debate.  oVirt is capable of providing the kinds of guarantees users need to run pet VMs while other projects are focused on cattle.
```

Comment:
I would prefer to avoid mention "pets" vs "cattle", as I find it has been really counterproductive in discussions - both oVirt and OpenStack are able to host either kind of workload.